packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -310,7 +314,9 @@ export class AppManagementClient implements DeveloperPlatformClient {
 
   async specifications({organizationId}: MinimalAppIdentifiers): Promise<RemoteSpecification[]> {
     const query = FetchSpecifications
-    const result = await appManagementRequestDoc(organizationId, query, await this.token())
+    const result = await appManagementRequestDoc(organizationId, query, await this.token(), undefined, {
+      cacheTTL: 6 * 60 * 60 * 1000,
```

Comment:
I believe we should add `organizationId` as an addendum to the cache key, as different organizations should have different results.

Reply:
The cache key is constructed as query+variables hash, so the orgId is included in the key in this case :)